manifests/0000_90_etcd-operator_03_prometheusrule.yaml

```diff
@@ -84,7 +84,7 @@ spec:
       expr: |
         histogram_quantile(0.99, rate(etcd_disk_backend_commit_duration_seconds_bucket{job=~".*etcd.*"}[5m]))
         > 0.25
```

Comment:
@wking Do you have any clever prom tricks that would maintain .25 for all platforms other than Azure and .5 for Azure?